ebusd-2.1.x/de/vaillant/15.700.csv

```diff
@@ -64,7 +64,7 @@ r,,HcStorageTempBottom,"Temp.fühler Hz, unten",,,,A000,,,tempv,,,"Temp.fühler
 # ##### Warmwasserkreis #####,,,,,,,,,,,,,
 *r,,,,,,B524,02000100,,,IGN:4,,,
 *w,,,,,,B524,02010100,,,,,,
-r;w,,HwcOpMode,HwcOpMode,,,,0300,,,opmode,,,operation mode of Hwc
+r;w,,HwcOpMode,Betriebsart Warmwasserkreis,,,,0300,,,opmode,,,operation mode of the domestic hot water circuit
```

Comment:
why have the field comment in en but the message comment in de?

Reply:
I have overlooked that. It should be "Operating mode of the hot water circuit".